client/containers/Users/Dialogs/PasswordResetDialog.jsx

```diff
@@ -33,12 +36,15 @@ export default connectContainer(class extends Component {
     cancelPasswordReset: PropTypes.func.isRequired,
     resetPassword: PropTypes.func.isRequired,
     connections: PropTypes.object.isRequired,
-    passwordReset: PropTypes.object.isRequired
+    passwordReset: PropTypes.object.isRequired,
+    appsForConnection: PropTypes.object
   };
 
   shouldComponentUpdate(nextProps) {
     return nextProps.passwordReset !== this.props.passwordReset ||
-      nextProps.languageDictionary !== this.props.languageDictionary
+      nextProps.languageDictionary !== this.props.languageDictionary ||
+      // nextProps.settings !== this.props.settings ||
```

Comment:
Did you mean to leave this comment?

Reply:
👋 Hi @yacineMTB , that's a part of the revert I made to the version with client_id selector: https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/200/files#diff-e88409dcf3bfb8a19e35f37b563d6a55L46